strongbox-security/strongbox-security-api/src/test/java/org/carlspring/strongbox/configuration/Base64PasswordEncoderDelegateTest.java

```diff
@@ -0,0 +1,71 @@
+package org.carlspring.strongbox.configuration;
+
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+
+@SpringBootTest
```

Comment:
it's seems that we don't need any Spring context here, you can just directly create `Base64PasswordEncoderDelegate` instance to test.